site/.gitignore

```diff
@@ -22,3 +22,4 @@
 /node_modules/*
 /web/build/*
 /yarn-error.log
+/assets/js/frontend-config.json
```

Comment:
Please do a separate PR for changes like this, they should be merged independent of this PR, which is more experimental.